### DISCUSSION TITLE
AE 里面设置PAG填充模式不生效

### DISCUSSION THREAD

**Discussion by xuboxi** [Tencent/libpag#1951] [General] [answered]
**【版本信息】**
操作系统：macos Ventura 13.5 ｜ PAGViewer 版本：4.3.6.102 ｜ AE版本：Version 23.6.0 (Build 62)

**【预期的表现】**
<img width="666" alt="image" src="https://github.com/Tencent/libpag/assets/19630247/7488bcfe-723e-407e-a924-638046d72880">
这里设置为Zoom后，导出后在PAG里更换图片依然还是LetterBox模式。

**Comment by jinwuwu001** [comment]
请问该问题是否还存在，若存在请提供下问题对应的aep和pag文件

**Comment by xuboxi** [comment]
邮件已收到，我会尽快回复。

**Comment by jinwuwu001** [comment]
您好，请提供下该问题对应的项目文件

**Comment by CodeJhF** [comment, marked as answer]
该问题已经由commit `d1f74a38937b7edb95dcad2e777e0db051e26fba` 修复，您可以更新libpag的版本试试，如果再发现问题，请反馈给我们。

**Comment by jinwuwu001** [comment]
该问题已修复，可从pag官网下载最新版PAGViewer，然后安装最新版AE导出插件
https://pag.io/

**Comment by xuboxi** [comment]
邮件已收到，我会尽快回复。
